app/models/weam.rb

```diff
@@ -165,12 +165,11 @@ def approved?
     return false unless poo_status =~ Regexp.new('aprvd', 'i')
 
     # VET TEC ONLY
-    return vet_tec_flags_for_approved? if applicable_law_code =~ Regexp.new("#{ALCVT}", 'i')
-    
+    return vet_tec_flags_for_approved? if applicable_law_code =~ Regexp.new(ALCVT.to_s, 'i')
```

Comment:
Where'd this come from?  Can we remove this change if it's not related?
Also, `ALCVT` _is_ a `string`, right?  Since it already is a string, you shouldn't need to call `to_s` or use string interpolation on it.

Reply:
It looks like both of these are auto-corrected by Rubocop

![image](https://user-images.githubusercontent.com/41960449/61663322-57d68e80-ac9e-11e9-882f-539fe006016b.png)

Reply:
I see.  We should have caught that when it was originally added.  
Anyway, I was saying since `ALCVT` is a string, you can just write:
```suggestion
    return vet_tec_flags_for_approved? if applicable_law_code =~ Regexp.new(ALCVT, 'i')
```
No `to_s` needed.  Rubocop will approve.